test/e2e/spec/e2e_spec.rb

```diff
@@ -1365,21 +1364,8 @@ def get_policy_id_from_decode(tx_decoded_mint_or_burn)
 
         tx_id = tx_submitted['id']
         wait_for_tx_in_ledger(@wid, tx_id)
-        src_after_minting = get_shelley_balances(@wid)
         target_after_minting = get_shelley_balances(@target_id)
 
-        # verify ADA balance is correct on src wallet:
-        # we are minting and sending to external address
-        # therefore the cost is fee + mintUTxOValue of ADA that is required
-        # for transfering the assets over the network
-        # in this the total `cost` is pure ADA minUTxOValue + 11 'utxo words' + fee
-        min_utxo_value = NETWORK.parameters['minimum_utxo_value']['quantity'].to_i
```

Comment:
Hi @piotr-iohk 

Just wondering: is it possible to hard-code the expected minimum UTxO value(s) in the test?

This might give us a sanity check that the values are reasonable. 🤔 

If the minimum UTxO value does change in any way (in future), then:
- it would cause this test to fail;
- we would have a chance to see the new value;
- we would have a chance to verify that it looks reasonable (i.e., to repeat the sanity check).

Alternatively, if it's too difficult, then I think deleting this is fair enough.

Reply:
Hm, actually at this moment is passes with this (i.e. with what GET network/parameters shows)... it failed last night, so I assume that it was due to the fact that there was different number of outputs in the transaction than usual (since the utxo was probably different). The wallet is out there on the testnet and it runs through ~200 tests every night, it is rather hard to keep the utxo under control.

So in other words, if minUtxOValue depends on the number of outputs in a transaction it is rather hard to come up with a value to hardcode it. (At least I don't have idea how).